src/controller/python/chip-device-ctrl.py

```diff
@@ -983,15 +993,21 @@ def zcl_add_network(node_id: int, ssid: str, password: str, endpoint_id: Optiona
     except Exception as e:
         return __get_response_dict(status = StatusCodeEnum.FAILED, error = str(e))
 
-def zcl_enable_network(node_id: int, ssid:str, endpoint_id: Optional[int] = 1, group_id: Optional[int] = 0, breadcrumb: Optional[int] = 0, timeoutMs: Optional[int] = 1000) -> Dict[str, Any]:
+def zcl_enable_network(node_id: int, args: Dict[str,str], endpoint_id: Optional[int] = 1, group_id: Optional[int] = 0, breadcrumb: Optional[int] = 0, timeoutMs: Optional[int] = 1000) -> Dict[str, Any]:
     try:
         __check_supported_os()
-        args = {}
-        args['networkID'] = ssid.encode("utf-8") + b'\x00'
-        args['breadcrumb'] = breadcrumb
-        args['timeoutMs'] = timeoutMs 
-  
-        err, res = device_manager.devCtrl.ZCLSend("NetworkCommissioning", "EnableNetwork", node_id, endpoint_id, group_id, args, blocking=True)
+        parameters = {}
+        if args.get("ssid"):
+            parameters['networkID'] = args.get("ssid").encode("utf-8") + b'\x00'
+        elif args.get("extpanid"):
+            parameters['networkID'] = bytes.fromhex(args.get("extpanid"))
+        else:
+            raise Exception("ssid or extpanid need to be provided")
+        parameters['breadcrumb'] = breadcrumb
+        parameters['timeoutMs'] = timeoutMs
+
+        err, res = device_manager.devCtrl.ZCLSend("NetworkCommissioning", "EnableNetwork", node_id, endpoint_id,
+                                                  group_id, parameters, blocking=True)
```

Comment:
line too long

Reply:
abbreviations should be avoided unless necessary for long names, use `error, result` instead.